README.md

```diff
@@ -166,14 +165,14 @@ If you don't have xclip or xsel, micro will use an internal clipboard for copy a
 
 ### Colors and syntax highlighting
 
-If you open micro and it doesn't seem like syntax highlighting is working, this is probably because
+If you open micro and find out that syntax highlighting is working, this is probably
```

Comment:
~~find out that syntax highlighting is working~~ → find out that syntax highlighting isn't working